### ISSUE TITLE
Show read request only on latest message

### ISSUE THREAD

**Issue by duarm** [Nheko-Reborn/nheko#1605]
### The Problem

we have 1 read icon  per-message, which doesn't make sense

![image](https://github.com/Nheko-Reborn/nheko/assets/26150581/97f9f9c2-8ade-4f0e-bd8a-1cb093c29ead)

### The Solution

only show 1 read icon, on your latest message

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by balejk** [comment]
I am in favour of leaving it as it is now (0.11.3) as that allows one to see *when* the given message (group) was viewed and by whom upon clicking the double check mark. It would however be useful if this was displayed for other people's messages as well in group chats for the same reason.

But sure, the UI could be different, at least in one-to-one chats the information mentioned above could be made accessible in a different way and there could then indeed be just one tick. On the other hand, I cannot think of how to generalize it to group chats without going all the way to how Element does it.

**Comment by balejk** [comment]
Looking at your screenshot now (I was only viewing the issue from CLI), those are in fact not read receipt icons but *Received* icons (I assume *received by the server*, so really rather *Sent*), which you can tell if you hover your mouse over them. Those it would perhaps really make sense not to display for each message or rather at all and just display the clocks when the message is being sent or the cross when sending fails.

**Comment by duarm** [comment]
Exactly, My wording was maybe not so great, but I was trying to describe what you said, which is also element's behaviour, you show a read receipt, or in case of element, the profile picture of each person who read that message, and only one of those can be active at the same time, so you don't get redundant information
